src/config/params.rs

```diff
@@ -140,3 +140,17 @@ fn color_to_solid_source(x: Color) -> SolidSource {
     let bytes = x.to_be_bytes();
     SolidSource::from_unpremultiplied_argb(bytes[3], bytes[0], bytes[1], bytes[2])
 }
+
+fn parse_cent(input: Option<&String>, default: u32) -> u32 {
+    let input: String = match input {
+        Some(a) => a.split_whitespace().collect(),
+        None => return default,
+    };
+    if input.contains('%') {
+        return match input.trim().strip_suffix('%').unwrap().parse::<u32>() {
```

Comment:
1. No need in `trim` there and below
2. `strip_prefix` returns `Option<&str>` without specified suffix, so you don't need `contains` call above